shuup/gdpr/resources.py

```diff
@@ -30,31 +34,51 @@ def valid_view(context):
     view_name = getattr(view_class, "__name__", "")
     if view_name == "EditorView":
         return False
+
     return True
 
 
 def add_gdpr_consent_resources(context, content):
     if not valid_view(context):
         return
 
-    shop = get_shop(context["request"])
+    request = context["request"]
+    shop = get_shop(request)
     gdpr_settings = GDPRSettings.get_for_shop(shop)
 
-    # GDPR not enabled
+    # GDPR not enabled, nothing to do
     if not gdpr_settings.enabled:
         return
 
+    # always add styles
+    add_resource(context, "head_end", static("shuup_gdpr/shuup_gdpr_styles.css"))
+
+    user = request.user
+    if not user.is_anonymous() and should_reconsent_privacy_policy(shop, user):
+        # TODO: Fix this as it may break when there are multiple consent documents
```

Comment:
I think this TODO should be handled as part of this, since we are not likely coming back to this anytime soon.